client/src/services/translation.ts

```diff
@@ -2,12 +2,30 @@
 
 import enUS from '@/locales/en-US.json';
 import frFR from '@/locales/fr-FR.json';
-import { InvitationStatus, WorkspaceRole } from '@/parsec';
+import { InvitationStatus, UserProfile, WorkspaceRole } from '@/parsec';
 import { DateTime } from 'luxon';
+import { App } from 'vue';
 import { createI18n } from 'vue-i18n';
 
 let i18n: any | null = null;
 
+export interface TranslationData {
+  key: string;
+  data?: object;
+  count?: number;
+}
+
+export type Translatable = string | TranslationData | undefined;
+
+export const TranslationPlugin = {
+  install: (app: App<any>): void => {
+    app.config.globalProperties.$msTranslate = (translatable: Translatable): string => {
+      return msTranslate(translatable);
+    };
+    app.provide('msTranslate', msTranslate);
```

Comment:
Better to define a key that can be used when injecting later rather than a raw string.